sdk/helper/certutil/helpers.go

```diff
@@ -510,6 +533,8 @@ func ValidateKeyTypeLength(keyType string, keyBits int) error {
 		default:
 			return fmt.Errorf("unsupported bit length for EC key: %d", keyBits)
 		}
+	case "ed25519":
+		return nil
```

Comment:
The return statement from here can be removed. Instead "ed25519" can be added along with "any" in the next case using a `,` separator.

Reply:
but that would cause something different. 
if i where to add it to the any it would print 
`return fmt.Errorf("unknown key type %s", keyType) ` 

Which is not the case and is why I added it separate.

Reply:
```
case "any", "ed25519":
default:
   return ...
```

☝️ should not print "unknown key type". It should satisfy the case statement and break out of the switch. The return at the end of the function should take effect.

Reply:
thanks didn't knew about this